frontend/src/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,136 @@
+import './SignUp.css'
+import React, { useContext, useState } from "react";
+import {useNavigate,Link,Router} from "react-router-dom"
+
+
+
+function SignUp(){
+    const navigate = useNavigate()
+    const [name, setName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [confirmPassword, setConfirmPassword] = useState('')
+    const [error, setError] = useState('')
+
+
+
+    async function handleSignUp(event){
+        event.preventDefault();
+
+        if (password !== confirmPassword) {
+            setError('Passwords do not match');
+            alert(error)
+            return;
+        }
+
+
+
+        try{
+            const response = await fetch("http://localhost:2500/signUp",{
+                method:"POST",
+                headers:{
+                    "Content-Type": "application/json",
+                },
+                body:JSON.stringify({
+                    name,
+                    email,
+                    password,
+                    confirmPassword,
+                }),
+                credentials: 'include'
+            });
+            console.log(response)
+            const data = await response.json()
+            if(response.ok){
+                goToHomepage()
+            } else{
+                setError('Registration failed')
+                alert(error)
+            }
+            if (data.error){
+                console.log(data.error)
```

Comment:
I think you can get rid of these console.log since you are already using `alert`